languages/JavaScript/exercises/concept/arrays-basic/.docs/after.md

```diff
@@ -0,0 +1,62 @@
+In Javascript, an array is actually just a regular object where the elements are properties of that object. It includes the `length` property and also has lots of useful methods for traversing and mutating the array.
+
+Accesing elements:
```

Comment:
Instead of listing all the various methods, I'd just link to documentation for methods. As an example, see this [C# example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/after.md).

What this document _should_ show is any syntax that is introduced. In this case, that means defining an array and accessing an array element using an indexer. So just having the following example code could replace the full listing:

```javascript
const names = ['Jack', 'Laura', 'Paul', 'Megan'];
names[1] = 'Lucy';
```